Document/0x06f-Testing-Local-Authentication.md

```diff
@@ -9,26 +9,196 @@ As a result only requirement "4.6	Biometric authentication, if any, is not event
 
 #### Overview
 
--- TODO [Provide a general description of the issue "Testing Biometric Authentication".]
+Biometric authentication on iOS is represented by Touch ID fingerprint sensing system. Touch ID sensor is operated by SecureEnclave<sup>[1]</sup> security coprocessor and do not expose fingerprint data to any other systems. With Touch ID set up, password is required only in certain cases (after 5 unsuccessful attempts, if device has been rebooted or was not unlocked in last 48 hours, etc), which encourages user to use longer and more complex passwords<sup>[2]</sup>.
 
-#### Static Analysis
-
--- TODO [Describe how to assess this given either the source code or installer package (APK/IPA/etc.), but without running the app. Tailor this to the general situation (e.g., in some situations, having the decompiled classes is just as good as having the original source, in others it might make a bigger difference). If required, include a subsection about how to test with or without the original sources.] --
-
--- TODO [Confirm purpose of remark "Use the &lt;sup&gt; tag to reference external sources, e.g. Meyer's recipe for tomato soup<sup>[1]</sup>."] --
+Third-party apps have two ways to incorporate system-provided Touch ID authentication:
```

Comment:
Maybe you can elaborate a bit about the differences between the two methods. Does the Security.Framework return data on the fingerprint? Is the check implemented in the application itself? Is it possible to bypass the check in any way? Is there a security difference between the two?